app/src/main/java/org/jellyfin/androidtv/data/service/BackgroundService.kt

```diff
@@ -145,6 +146,28 @@ class BackgroundService(
 		val parentBackdropUrls = baseItem.parentBackdropImageTags.getUrls(baseItem.parentBackdropItemId)
 		val backdropUrls = itemBackdropUrls.union(parentBackdropUrls)
 
+		loadBackgrounds(backdropUrls)
+	}
+
+	/**
+	 * Use backdrop from [SearchHint] as background.
+	 */
+	fun setBackground(searchHint: SearchHint?) {
```

Comment:
Does this actually need to be nullable? All items in the search results are searchhints so they can't be null.

Reply:
Sure doesn't lol